test_data/asm/vm_to_vm_dynamic_trace_length.asm

```diff
@@ -0,0 +1,70 @@
+machine Main {
+
+    degree 256;
+
+    Pow pow;
+
+    reg pc[@pc];
+    reg X[<=];
+    reg Y[<=];
+    reg Z[<=];
+    reg A;
+
+    instr pow X, Y -> Z = pow.pow
+    instr assert_eq X, Y { X = Y }
+
+    function main {
+        A <== pow(7, 0);
+        assert_eq A, 1;
+
+        A <== pow(7, 1);
+        assert_eq A, 7;
+
+        A <== pow(7, 2);
+        assert_eq A, 49;
+
+        A <== pow(2, 2);
+        assert_eq A, 4;
+
+        A <== pow(2, 10);
+        assert_eq A, 1024;
```

Comment:
Awesome! Can you add a few calls that would overflow? Is that then mod GL?

Reply:
Yes, all of this is field arithmetic. I could add an overflowing example, but then we couldn't test this VM in both the GL and BN254 field (at least not with the same assertions). I don't really see the point TBH, what are you trying to test?

Reply:
just trying to get some coverage and reminding whoever reads this of that, or whenever something changes wrt a field, something should happen.

But yea fine with keeping it as is